src/app/beer_garden/router.py

```diff
@@ -471,7 +471,14 @@ def _pre_route(operation: Operation) -> Operation:
             operation.model.namespace = config.get("garden.name")
 
     elif operation.operation_type == "SYSTEM_READ_ALL":
-        if operation.kwargs.get("filter_params", {}).get("namespace", "") == "":
+        # what looks like a sensible edit is to change the next line to:
+        #   operation.kwargs.get("filter_params", {}).get("namespace", "") == "":
+        # but that will break everything, as it turns out. So, unless
+        # operation.kwargs has a key called filter_params whose value is a dictionary
+        # that has a key called namespace and its value is an empty string, the follow-
+        # ing never runs. Intuition says that's not what was intended here, so
+        # TODO: look into this
+        if operation.kwargs.get("filter_params", {}).get("namespace") == "":
```

Comment:
The reason this change breaks everything is that it was now the case that if you defined no filter params at all, we would effectively apply a default filter limiting it to the local garden only.  This is clearly different behavior than before, where it would only default if we got here and a namespace filter was present, but set to empty string.

I agree that it is not obvious from the code **why** someone would supply a filter of `namespace = ""`, nor why the expected behavior in that case would be that we treat that as `namespace = <local garden>`.  It wouldn't surprise me at all if that behavior isn't in fact desired, but it didn't matter because it's never induced (i.e. nothing actually gets us here with the `namespace = ""` filter param).